### ISSUE TITLE
Pandas dataframe of 2220 recordings containing urls cannot be opened

### ISSUE THREAD

**Issue by youssef-zied** [spyder-ide/spyder#16954]
### What steps will reproduce the problem?

Pandas dataframe of 2220 recordings containing urls cannot be opened

### Traceback
```python-traceback
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\variableexplorer\widgets\collectionseditor.py", line 1038, in copy
    obj = self.delegate.get_value(idx)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\variableexplorer\widgets\collectionseditor.py", line 1373, in get_value
    return self.parent().get_value(name)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\variableexplorer\widgets\collectionseditor.py", line 1431, in get_value
    value = self.shellwidget.get_value(name)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole\widgets\namespacebrowser.py", line 97, in get_value
    return self.call_kernel(
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 549, in __call__
    return self._comms_wrapper._get_call_return_value(
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole\comms\kernelcomm.py", line 151, in _get_call_return_value
    return super(KernelComm, self)._get_call_return_value(
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 432, in _get_call_return_value
    return self._sync_error(reply['value'])
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 488, in _sync_error
    error_wrapper.raise_error()
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 90, in raise_error
    raise self.etype(self)
Exception in comms call get_value:

  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 344, in _handle_remote_call
    self._set_call_return_value(msg_dict, return_value)

  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 376, in _set_call_return_value
    self._send_message('remote_call_reply', content=content, data=data,

  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder_kernels\comms\commbase.py", line 244, in _send_message
    buffers = [cloudpickle.dumps(

  File "C:\ProgramData\Anaconda3\lib\site-packages\cloudpickle\cloudpickle_fast.py", line 72, in dumps
    cp.dump(obj)

  File "C:\ProgramData\Anaconda3\lib\site-packages\cloudpickle\cloudpickle_fast.py", line 540, in dump
    return Pickler.dump(self, obj)

TypeError: cannot pickle '_thread.lock' object

```

## Versions

* Spyder version: 4.1.4 
* Python version: 3.8.3
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.5.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.16.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.5.3 (OK)
pyls >=0.34.0;<1.0.0           :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.5 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.1.2 (OK)
spyder_kernels >=1.9.2;<1.10.0 :  1.9.2 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  0.29.21 (OK)
matplotlib >=2.0.0             :  3.4.3 (OK)
numpy >=1.7                    :  1.18.5 (OK)
pandas >=0.13.1                :  1.0.5 (OK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  1.6.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @youssef-zied, thanks for reporting. I think this problem is fixed in our latest version available in Anaconda (5.1.5). Please update by closing Spyder, opening the Anaconda Prompt and running there

    conda remove spyder
    conda remove python-language-server
    conda update anaconda
    conda install spyder=5.1.5

Don't worry if some of those commands give you errors, just continue with the rest.

Let us know if that works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
